docs/sanic/request_data.md

```diff
@@ -120,3 +120,19 @@ args.get('titles') # => 'Post 1'
 
 args.getlist('titles') # => ['Post 1', 'Post 2']
 ```
+
+## Accessing the handler name with the request.endpoint attribute
+
+The `request.endpoint` attribute holds the handler's name. For instance, the below
+route will return "hello".
```

Comment:
I think in a blueprint, it should also include the blueprint's name? For example: `foo.hello`, `bar.hello`

Reply:
I like that idea